com.unity.render-pipelines.universal/Runtime/Passes/ScriptableRenderPass.cs

```diff
@@ -189,6 +193,8 @@ public ScriptableRenderPass()
             overrideCameraTarget = false;
             isBlitRenderPass = false;
             profilingSampler = new ProfilingSampler(nameof(ScriptableRenderPass));
+            useNativeRenderPass = true;
```

Comment:
This API is internal, what does it mean for custom passes that can't change this userNativeRenderPass?

Reply:
TODO: add a comment

Reply:
At the end, we expect to have a preprocessing mechanism that would determine whether pass is eligible or not to be inside a RenderPass, so this is temporary. Also, Native RenderPass has to be enabled in the renderer asset and having this value as true won't really affect anything unless it's explicitly enabled